source/TestState.hx

```diff
@@ -44,25 +44,25 @@ class TestState extends FlxState {
 		}
 	}
 
-	private function setUpPursuitTank1(targetTank) {
-		var pursuitTank = new Tank(100, 100);
-		pursuitTank.setControllers(new PursuePlayerMovementController(pursuitTank, targetTank),
-			new SpinShootController(pursuitTank));
-		add(pursuitTank.getAllSprites());
-	}
+	/*private function setUpPursuitTank1(targetTank) {
+			var pursuitTank = new Tank(100, 100);
+			pursuitTank.setControllers(new PursuePlayerMovementController(pursuitTank, targetTank),
+				new SpinShootController(pursuitTank));
+			add(pursuitTank.getAllSprites());
+		}
 
-	private function setUpPursuitTank2(targetTank) {
-		var pursuitTank = new Tank(300, 400);
-		pursuitTank.setControllers(new PursuePlayerMovementController(pursuitTank, targetTank),
-			new AutoAimController(pursuitTank, targetTank));
-		add(pursuitTank.getAllSprites());
-	}
+		private function setUpPursuitTank2(targetTank) {
+			var pursuitTank = new Tank(300, 400);
+			pursuitTank.setControllers(new PursuePlayerMovementController(pursuitTank, targetTank),
+				new AutoAimController(pursuitTank, targetTank));
+			add(pursuitTank.getAllSprites());
+		}
 
-	/*private function setUpDynamicTank(targetTank) {
-		var dynamicTank = new Tank(50, 300);
-		dynamicTank.setControllers(new DynamicMovementController(dynamicTank, targetTank),
-			new AutoAimController(dynamicTank, targetTank));
-		add(dynamicTank.getAllSprites());
+		private function setUpDynamicTank(targetTank) {
+			var dynamicTank = new Tank(50, 300);
+			dynamicTank.setControllers(new DynamicMovementController(dynamicTank, targetTank),
+				new AutoAimController(dynamicTank, targetTank));
+			add(dynamicTank.getAllSprites());
 	}*/
```

Comment:
In general, it's a good idea to not leave swaths of code commented out in the things we merge. This is a bit of a special case, since the code in question is a "test" file that is kinda intended to be messy from the outset.

No change needed, but figured it'd be worth keeping in mind for future projects or situations not involving messy test files like this.